Big-Spender/readme.md

```diff
@@ -68,39 +68,41 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+select * from spends where description ilike '%fee%';
 ```
 
 **Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+select * from spends where expense_area_id = 2;
 ```
 
 **Claire:** Great, that's very helpful. How about the total amount spent for each month?
 
 **You:** You can get that by using the GROUP BY clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+SELECT sum(amount) AS total_monthly_amount, date AS month from spends group by month;
```

Comment:
I don't think this query is accurate, it doesn't show the total amount by month, it shows the total amount by date. The date can have a day, or year as well.

You need to extract the month portion of the date and group by that.

Take a look at https://www.postgresqltutorial.com/postgresql-date-functions/postgresql-extract/ for one way on how to do that.